website/blog/2023-06-14-install-ferretdb-mongodb-alternative-ubuntu.mdx

```diff
@@ -0,0 +1,321 @@
+
+```sh
+sudo apt update
+sudo apt upgrade
+```
+
+Once the system has been updated, install PostgreSQL
+
+```sh
+sudo apt install postgresql -y
+```
+
+After installing, ensure that the service is runnning:
+
+```sh
+$ systemctl status postgresql
+● postgresql.service - PostgreSQL RDBMS
+     Loaded: loaded (/lib/systemd/system/postgresql.service; enabled; vendor preset: enabled)
+     Active: active (exited) since Fri 2023-04-14 18:23:20 EAT; 1min 7s ago
+    Process: 4519 ExecStart=/bin/true (code=exited, status=0/SUCCESS)
+   Main PID: 4519 (code=exited, status=0/SUCCESS)
+        CPU: 1ms
+....
+```
+
+Connect to the instance:
+
+```sh
+sudo -u postgres psql
+```
+
+Create the FerretDB user:
+
+```js
+CREATE USER ferretuser WITH PASSWORD 'Passw0rd!';
+```
+
+Create a database for the user:
+
+```sql
+CREATE DATABASE ferretuser OWNER ferretuser;
+```
+
+Once created, exit the shell:
+
+```sh
+\q
+```
+
+Create the user on your system:
+
+```sh
+sudo adduser ferretuser
+```
+
+Verify if you can connect to the database using the user:
+
+```sh
+$ sudo -u ferretuser psql
+could not change directory to "/home/ubuntu22": Permission denied
+psql (14.7 (Ubuntu 14.7-0ubuntu0.22.04.1))
+Type "help" for help.
+
+ferretuser=> \q
+```
+
+## Step 2 – Download and Install FerretDB on Ubuntu
+
+To download the latest FerretDB release, visit the official [GitHub Release page](https://github.com/FerretDB/FerretDB/releases).
+Alternatively, you can pull the latest release using the command:
+
+```sh
+VER=v1.2.0
+wget https://github.com/FerretDB/FerretDB/releases/download/$VER/ferretdb.deb
+```
+
+Once downloaded, install FerretDB using the command:
+
+```sh
+sudo apt install ./ferretdb.deb
+```
+
+Sample Output:
+
+```sh
+Reading package lists... Done
+Building dependency tree
+Reading state information... Done
+Note, selecting 'ferretdb' instead of './ferretdb.deb'
+The following package was automatically installed and is no longer required:
+  gir1.2-goa-1.0
+Use 'sudo apt autoremove' to remove it.
+The following NEW packages will be installed:
+  ferretdb
+0 upgraded, 1 newly installed, 0 to remove and 95 not upgraded.
+Need to get 0 B/15.1 MB of archives.
+After this operation, 30.1 MB of additional disk space will be used.
+Get:1 /home/ubuntu/ferretdb.deb ferretdb amd64 0.0.0~rc0 [15.1 MB]
+Selecting previously unselected package ferretdb.
+(Reading database ... 234140 files and directories currently installed.)
+Preparing to unpack /home/ubuntu/ferretdb.deb ...
+Unpacking ferretdb (0.0.0~rc0) ...
+Setting up ferretdb (0.0.0~rc0) ...
+```
+
+Check the installed version:
+
+```sh
+$ ferretdb --version
+version: v1.2.0
+commit: 3153c8fbf185126af1fe8fb364ac166d2287d093
+branch: unknown
+dirty: true
+package: deb
+debugBuild: false
+```
+
+## Step 3 – Start and Enable FerretDB on Ubuntu
+
+FerretDB takes several arguments that include:
```

Comment:
Please actually run FerretDB with `--help` flag to see and document the actual flags, not just try to guess what is wrong there. There were a few formatting issues that seem to be fixed manually, and now the result is incorrect. It might be easier to replace the whole block with the actual output.